carbons/carbons.go

```diff
@@ -61,3 +61,65 @@ func DisableIQ(ctx context.Context, s *xmpp.Session, iq stanza.IQ) error {
 	)), &v)
 	return err
 }
+
+
+//DisableCarbon is an xmlstream.Transformer that inserts <private xmlns='urn:xmpp:carbons:2'/>
+//into an individual message, which in turn disables carbons for that message. If a message contains
+//<private xmlns='urn:xmpp:carbons:2'/> already then we will not add it. Other we will add it to everything
+//for now and sort it out later
+
```

Comment:
For this to show up as the help text there can't be any whitespace between it and the function definition, please remove these two empty lines.